### ISSUE TITLE
ATOM-ONE Hub communications

### ISSUE THREAD

**Issue by COverS8** [atomone-hub/genesis#21]
With ATOM-ONE should be considered better alternatives for communications than Telegram which is not end-2-end encrypted and is full of malicious phishing bots or Discord which have suffered from hackings. Some better secure app where the official governance with different channels could be created

**Comment by jaekwon** [comment]
I think this should be generally incubated/funded within the scope of the "Decentralists" which is a group of people that are aligned with AtomOne, but also we should be funding UX/tooling/social-communication-systems for the Decentralists that are secure. 

Absolutely. If not the Decentralists, this can still be done as a community pool funded endeavor; but the constitution should make clear that this sort of work is allowed; and also specify limitations, like we cannot require users to use any of our tools, especially if they aren't suitable in the end despite our efforts.

It's not clear to me how best it should be incorporated yet so let's keep discussing here.

**Comment by FarOutAndCosmic** [comment]
Agreed regards Telegram. I'm wondering if something like Desmos with a combination of LIKE coin could be utilised? Showing proactive support to existing cosmos ecosytem products may well drive good sentiments all round.. Plus we get to track input from the community..
